upgrade/timeLockUpgrade.js

```diff
@@ -54,22 +55,24 @@ async function main() {
         let newImplPolygonAddress;
 
         if (upgrade.constructorArgs) {
-            newImplPolygonAddress = await upgrades.prepareUpgrade(proxyPolygonAddress, polygonZkEVMFactory,
+            newImplPolygonAddress = await upgrades.prepareUpgrade(
+                proxyPolygonAddress,
+                polygonZkEVMFactory,
                 {
                     constructorArgs: upgrade.constructorArgs,
                     unsafeAllow: ['constructor', 'state-variable-immutable'],
                 },
             );
 
             console.log({ newImplPolygonAddress });
-            console.log("you can verify the new impl address with:")
+            console.log('you can verify the new impl address with:');
             console.log(`npx hardhat verify --constructor-args upgrade/arguments.js ${newImplPolygonAddress} --network ${process.env.HARDHAT_NETWORK}\n`);
-            console.log("Copy the following constructor arguments on: upgrade/arguments.js \n", upgrade.constructorArgs)
+            console.log('Copy the following constructor arguments on: upgrade/arguments.js \n', upgrade.constructorArgs);
```

Comment:
Wouldn't this print a literal \n when we want a new line?

Reply:
In my terminal, it prints a new line.